packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -423,21 +424,16 @@ export const signTransaction =
             enhancedTxInfo.useDecreaseOutput = hasDecreasedOutput;
         }
 
-        if (
-            account.networkType === 'ethereum' &&
-            !isCardanoTx(account, enhancedTxInfo) &&
-            enhancedTxInfo.token?.contract &&
-            network?.chainId
-        ) {
-            const isTokenKnown = await fetch(
-                `https://data.trezor.io/firmware/eth-definitions/chain-id/${
-                    network.chainId
-                }/token-${enhancedTxInfo.token.contract.substring(2).toLowerCase()}.dat`,
-            )
-                .then(response => response.ok)
-                .catch(() => false);
-
-            enhancedTxInfo.isTokenKnown = isTokenKnown;
+        if (account.networkType === 'ethereum' && !isCardanoTx(account, enhancedTxInfo)) {
+            const contractAddress = enhancedTxInfo.token?.contract;
+            if (contractAddress) {
```

Comment:
If we will check the existence of contract address in the condition above this is not needed then

Reply:
52ef3043a0d6e73f882cbc48a2a12690e521897f